### DISCUSSION TITLE
How to interpret the trained model?

### DISCUSSION THREAD

**Discussion by landkwon94** [ourownstory/neural_prophet#1382] [Q&A - get help using NeuralProphet]
Hello Sir!

First of all, thank you so much for sharing your amazing codes!

I am using NeuralProphet model for train/validate/test for time-series forecasting.

After training the model, I want to visualize 'feature importance' or 'critical time period'.

As the original paper title is 'NeuralProphet: Explainable Forecasting at Scale', I hope there are some modules for explaining the model forecasting.

I want to know whether it is possible, and how can it be performed!

I will wait for your reply :) 

Many thanks for your contributions :)

**Comment by leoniewgnr** [comment]
Hi @landkwon94, what exactly do you mean with 'feature importance' or 'critical time period'?
NP is interpretable because you can decompose the forecast into its single components: trend, seasonality, AR, Future and Lagged regressors and events. To further dive into the interpretability, I can recommend to check out this [tutorial 2-7](https://neuralprophet.com/tutorials/index.html) and especially for visualization this [feature guide](https://neuralprophet.com/how-to-guides/feature-guides/plotly.html).  Let me know if you have any further questions
